spec/filters/bugs/pack_unpack.rb

```diff
@@ -82,4 +82,29 @@
   fails "Array#pack with format 'l' with modifier '>' encodes the number of array elements specified by the count modifier" # RuntimeError: Unsupported pack directive "l>" (no chunk reader defined)
   fails "Array#pack with format 'l' with modifier '>' ignores NULL bytes between directives" # RuntimeError: Unsupported pack directive "l>" (no chunk reader defined)
   fails "Array#pack with format 'l' with modifier '>' ignores spaces between directives" # RuntimeError: Unsupported pack directive "l>" (no chunk reader defined)
+  fails "Array#pack with format 'U' raises a TypeError if #to_int does not return an Integer" # Expected TypeError but no exception was raised ("\u0005" was returned)
```

Comment:
This spec says that there must be a `Opal.coerce_to! :to_int` call on each element of the array, could you add it please? Just for reference: there are a lot of such calls in corelib.

Reply:
I think I'm already calling the `ToInt` function on each element.